ChangeLog.md

```diff
@@ -1,5 +1,162 @@
 # Changelog for haskell-language-server
 
+## 1.4.0
+
+After a month of vacation a new hls release has arrived:
+
+- Support for ghc 8.10.6 and 8.10.7
+- The ormolu formatter plugin works with ghc 9.0.1
+- We have a *new brand plugin to let you rename indentifiers* thanks to @OliverMadine's great work
+![rename-demo](https://user-images.githubusercontent.com/30090176/133071923-b3d91b2a-619f-4151-ab76-ca50442e5ab4.gif)
```

Comment:
The plugin is disabled in Cabal so it won't be possible to enable it for binary users. This needs to be highlighted

Reply:
oh, didnt notice that :worried:

Reply:
So i think it is better to remove it from the description directly, as advanced users enabling it on purpose already are aware

Reply:
Removed and opened #2193

Reply:
@pepeiborra are you fine with the remove?

Reply:
Sure, I respect @OliverMadine choice

Reply:
It seems reasonable to me to remove it until we have the multi-component support.

Reply:
ok so it only needs a stamp to go ☺️

Reply:
@pepeiborra does it needs some other change?